src/Common/src/Interop/Windows/mincore/Interop.GetFullPathNameW.cs

```diff
@@ -22,19 +22,25 @@ partial class mincore
         [DllImport(Libraries.CoreFile_L1, EntryPoint = "GetFullPathNameW", SetLastError = true, CharSet = CharSet.Unicode, BestFitMapping = false)]
         private static extern int GetFullPathNamePrivate(string path, int numBufferChars, [Out]StringBuilder buffer, IntPtr mustBeZero);
 
-        internal static int GetFullPathName(string path, int numBufferChars, [Out]StringBuilder buffer, IntPtr mustBeZero)
+        internal static int GetFullPathName(string path, int numBufferChars, [Out]StringBuilder buffer)
```

Comment:
> Once we start letting arbitrary changes in to these  Interop.*.cs  files, they lose their meaning.

I agree that we shouldn't let arbitrary changes in- but these are specific changes. Wrapping that ensures correct and/or safe usage of the underlying API is within scope of what the interop layer should have.

As I said, I can't understand what value there in forcing this code into another layer is beyond aesthetics. I can, however, easily fathom how having the correctness method outside of interop would introduce nasty and subtle bugs- so I'm seriously uncomfortable doing so.